tests/utils/argparse_test.py

```diff
@@ -163,3 +163,18 @@ def test_fifo(self):
             path = os.path.join(d, 'fifo')
             os.mkfifo(path)
             cargparse.optional_file_like(path)
+
+
+class TestIntOrPlusOrMinus(unittest.TestCase):
+    """Tests for the is_int_or_pm validator."""
+
+    def test_int(self):
+        cargparse.is_int_or_pm('5')
```

Comment:
I think this should be `self.assertTrue(cargparse.is_int_or_pm('5'))`, and for `test_pm` too.

Reply:
oh ok? I honestly just copied the test just above, check e.g. line 159 in that file.
So should we adjust those as well?

Reply:
optional_file_like doesn't have a return value, it either raises an exception or it doesn't. So I think those tests are correct.

Reply:
ok. Right: so my validator has a return value but (as mentioned below) it will never be used. So maybe the name `is_..` is misleading and one could think that it would return a bool..